.github/workflows/inspektor-gadget.yml

```diff
@@ -239,15 +229,9 @@ jobs:
         registry: ${{ env.REGISTRY }}
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
-    - name: Set IMAGE_TAG
-      run: |
-        TMP1=${GITHUB_REF#*/}
-        TMP2=${TMP1#*/}
-        IMAGE_TAG=${TMP2//\//-}
-        if [ "$IMAGE_TAG" = "main" ]; then
-            IMAGE_TAG="latest"
-        fi
-        echo IMAGE_TAG=$IMAGE_TAG >> $GITHUB_ENV
+    - name: Craft container image tag
+      id: craft-image-tag
+      uses: ./.github/actions/craft-image-tag
     - name: Choose container repository
       id: choose-repo
       uses: ./.github/actions/choose-container-repo
```

Comment:
I thought the same as @mauriciovasquezbernal but I forgot to mentioned it. Why not to combine these two custom actions? That was also my suggestion [here](https://github.com/kinvolk/inspektor-gadget/pull/743#discussion_r929695959).